numpy/f2py/setup.py

```diff
@@ -39,8 +39,8 @@ def configuration(parent_package='', top_path=None):
     config = configuration(top_path='')
     config = config.todict()
 
-    config['download_url'] = "http://cens.ioc.ee/projects/f2py2e/2.x"\
-                             "/F2PY-2-latest.tar.gz"
+    config['download_url'] = "https://web.archive.org/web/20140822061353/"\
+        "http://cens.ioc.ee/projects/f2py2e/2.x/F2PY-2-latest.tar.gz"
```

Comment:
Not sure we need to provide this; am ambivalent about removal though.

Reply:
I think it is outdated. @pearu comment?

Reply:
Thinking about it more, I'd just remove it. F2PY is now distributed with NumPy and has undergone significant changes.